cmd/gaiad/cmd/root.go

```diff
@@ -122,6 +128,8 @@ func queryCommand() *cobra.Command {
 		rpc.BlockCommand(),
 		authcmd.QueryTxsByEventsCmd(),
 		authcmd.QueryTxCmd(),
+		feegrantcli.GetQueryCmd(),
+		authzcli.GetQueryCmd(),
```

Comment:
Do we need to explicitly add these? The module manager in app.go takes care of this for us.

Reply:
let's remove these @yaruwangway